src/views/List.jsx

```diff
@@ -11,36 +12,47 @@ export function List({ data }) {
 	const filteredItems = data.filter((item) =>
 		item.name.toLowerCase().includes(searchItem.toLowerCase()),
 	);
-
 	return (
 		<>
-			<form onSubmit={(event) => event.preventDefault()}>
-				<label htmlFor="search-item">Search Item: </label>
+			<p>Welcome to the {listPath.slice(29)} list!</p>
+
```

Comment:
I really like the welcoming message! I only wonder whether it should be welcoming a user every time they open the list?

Reply:
moved the welcome message to only show up when the list is empty